### DISCUSSION TITLE
Error message at startup : An unexpected error occurred when locating the Git executable

### DISCUSSION THREAD

**Discussion by AlexandreBourrieau** [eclipse-theia/theia#10606] [General]
Hello,

When I launch theia with the command : 

/home/user/.nvm/versions/node/v12.22.7/bin/yarn theia start /home/user/catkin_ws/ --hostname 0.0.0.0 --port 8000

I got the previous error message when connecting with my browser.

Do you know how to remove it ?

Thanks a lot !

**Comment by colin-grant-work** [comment]
@AlexandreBourrieau, can you say a little more about your set up? It looks like you're running in browser mode locally or a POSIX system? Is that correct? Do you have `git` installed and is it available on your path?

**Comment by AlexandreBourrieau** [comment]
Hello,

I use ubuntu 20.04.

I tried to add PATH=$PATH:/usr/bin/ in .env file in my worspace and to add GIT_EXEC_PATH=/usr/bin/git but I still get the error message...

**Reply by vince-fugnitto** [reply]
@AlexandreBourrieau in your case is there any loss of functionality, or is it simply an error message? Do you happen to open a workspace and if so is it under git version control?

I found some related discussons/issues:
- https://github.com/arduino/arduino-ide/issues/163
- https://github.com/arduino/arduino-pro-ide/issues/30

**Reply by AlexandreBourrieau** [reply]
Yes all is working great, it's just the message at startup like in the Arduino. Not very important just a little annoying.
